test/data/xy_rate.yaml

```diff
@@ -0,0 +1,103 @@
+        unit: mA/cm-2 # uA, mA, A, mA/cm-2, uA/cm-2, A/cm-2, A/m-2, ...
+    linked measurements: null # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
+    comment: # Add whatever you think is important to understand the data
+
+electrochemical system:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: NaCl # replace name by something generic, i.e., for a complex organic compound
+              sum formula:  NaCl
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: salt #
+              source:
+                  supplier: null
+                  LOT: 
+            - name: water
+              sum formula: H2O
+              type: solvent
+              proportion:
+                  value: 50
+                  unit: volume percent
+              source:
+                  refinement: Sartorius Arium 611UV
+                  quality: ultrapure water
+                  total organic carbon: 
+                      value: 3
+                      unit: ppb
+            - name: ethanol
+              sum formula: EtOH
+              type: solvent
+              proportion:
+                  value: 50
+                  unit: volume percent
+            - name: sulfuric acid
+              sum formula: H2SO4
+              type: acid
+              source:
+                  supplier: null
+                  LOT: 
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value: 13
+            uncertainty: 0.1
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: three # number of electrodes
+        reference electrode:
+            type: RHE # the reference electrode given in the experimental section. Might differ from that in the plot: MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier: SI Analytics
+        counter electrode:
+            material: Pt
+            crystallographic orientation: poly
+            shape: wire
+        working electrode:
+            material: Ni
+            crystallographic orientation: 111 # hkl, 100, 110, poly
+            source:
+                supplier: null
+                LOT: 
+            shape: # optional
+                type: cylinder # bead/sphere
+                height:
+                    value: 4
+                    unit: mm
+                diameter:
+                    value: 4
+                    unit: mm
+            geometric electrolyte contact area: 
+                value: null
```

Comment:
Do we want to support `null + unit`? Or should this say `0 + unit`?

Reply:
We thought `null` was a good option when the value was unknown.